seed/static/seed/partials/mapping_xml.html

```diff
@@ -0,0 +1,225 @@
+              <div class="alert alert-danger" ng-show="!required_taxlot_fields_present() && !import_file.matching_done" translate>At least one of the following Tax Lot fields is required: Address Line 1, Custom ID 1, Jurisdiction Tax Lot ID, or ULID.</div>
+              <div class="alert alert-danger" ng-show="duplicates_present" translate>Please review SEED headers.  Duplicate values are not allowed.</div>
+              <div class="alert alert-danger" ng-show="empty_fields_present()" translate>Please review SEED headers.  Empty values are not allowed.</div>
+              <table class="table table-striped" ng-hide="review_mappings">
+                  <thead>
+                      <tr>
+                          <th colspan="3" class="source_data">SEED</th>
+                          <th colspan="6" class="source_data">{$:: import_file.uploaded_filename $}</th>
+                      </tr>
+                  </thead>
+                  <tbody>
+                      <tr>
+                          <td style="border-right: 0 none; max-width: 160px; width: 160px; text-align: right;">
+                              <!-- <span style="padding-right:5px;" ng-hide="import_file.matching_done">{$:: 'Set all fields to' | translate $}:</span>
+                              <select ng-model="setAllFields" ng-change="setAllInventoryTypes()" ng-options="item as (item.name | translate) for item in ::setAllFieldsOptions track by item.value" ng-hide="import_file.matching_done"></select> -->
+                          </td>
+                          <td style="border-right: 0 none;"><h3 translate>Mapped Fields</h3></td>
+                          <td style="max-width: 128px; width: 128px;"></td>
+                          <td colspan="6"></td>
+                      </tr>
+                  </tbody>
+                  <thead>
+                      <tr>
+                          <th><translate>Inventory Type</translate></th>
+                          <th class="mapping_field ellipsis-resizable" sd-resizable><translate>SEED Header</translate></th>
+                          <th><translate>Measurement Units</translate></th>
+                          <th class="import_column_name ellipsis-resizable" sd-resizable><translate>Data File Header</translate></th>
+                      </tr>
+                  </thead>
+                  <tbody id="mapped-table">
+                      <tr ng-repeat="col in mappings">
+                          <td style="text-align: right;" ng-class="{'danger': col.is_duplicate || col.suggestion === ''}" ng-attr-id="mapped-row-type-{$:: $index $}">
+                              <select ng-model="col.suggestion_table_name" ng-change="updateInventoryTypeDropdown(); change(col)" ng-disabled="true">
```

Comment:
I reviewed this thinking that this file is basically a copy of the existing mapping partial except the ng-disabled="true" was added on these table items. 

Given we're not sure what'll need later, so I think keeping all of this is fine.